apps/wallet/src/ui/app/components/ledger/useDeriveLedgerAccounts.ts

```diff
@@ -73,8 +80,10 @@ async function deriveAccountsFromLedger(
         );
         const publicKey = new Ed25519PublicKey(publicKeyResult.publicKey);
         ledgerAccounts.push({
+            type: AccountType.LEDGER,
+            address: `0x${publicKey.toSuiAddress()}`,
```

Comment:
Is the address always non 0x prefixed?I wish our APIs were more consistent here.

Reply:
Let me use `normalizeSuiAddress` here! I didn't realize that existed even though I was using it the part 1 PR haha